conf/portal.application.conf

```diff
@@ -231,6 +234,14 @@ expressionRepository {
   expressionQueryGenerator.type = "com.arpnetworking.metrics.portal.expressions.impl.DatabaseExpressionRepository$GenericQueryGenerator"
 }
 
+# Rollups
+# ~~~~~
+rollups {
+  fetch.interval = "1h"
+  metric.whitelist = [""]
```

Comment:
Can you make these empty lists instead of a single element list with an empty string?

Reply:
Oops, that's what I meant, just spaced out a little.